src/redux/slices/editingV2Project.ts

```diff
@@ -49,44 +49,39 @@ export type NftRewardsData = {
   postPayModal: NftPostPayModalConfig | undefined
 }
 
-interface V2ProjectState {
-  version: number
+interface CreateState {
+  fundingCyclesPageSelection: 'automated' | 'manual' | undefined
+  reconfigurationRuleSelection: ReconfigurationStrategy | undefined
+  createFurthestPageReached: CreatePage
+  createSoftLockPageQueue: CreatePage[] | undefined
+  fundingTargetSelection: FundingTargetType | undefined
+  payoutsSelection: PayoutsSelection | undefined
+  projectTokensSelection: ProjectTokensSelection | undefined
+}
+
+interface ProjectState {
   projectMetadata: ProjectMetadataV5
   fundingCycleData: SerializedV2V3FundingCycleData
   fundingCycleMetadata: SerializedV2V3FundingCycleMetadata
   fundAccessConstraints: SerializedV2V3FundAccessConstraint[]
-  fundingTargetSelection: FundingTargetType | undefined
   payoutGroupedSplits: ETHPayoutGroupedSplits
-  payoutsSelection: PayoutsSelection | undefined
   reservedTokensGroupedSplits: ReservedTokensGroupedSplits
-  projectTokensSelection: ProjectTokensSelection | undefined
   nftRewards: NftRewardsData
-  fundingCyclesPageSelection: 'automated' | 'manual' | undefined
-  reconfigurationRuleSelection: ReconfigurationStrategy | undefined
-  createFurthestPageReached: CreatePage
-  createSoftLockPageQueue: CreatePage[] | undefined
   mustStartAtOrAfter: string
 }
 
+type ReduxState = {
+  version: number
+} & ProjectState &
+  CreateState
```

Comment:
I think ideally we should create clearer boundaries between create-flow data and generic project data. Redux is used in project reconfiguration too, where `CreateState` properties aren't relevant. Perhaps we should even have an entirely separate redux store for the create flow.

Reply:
love this idea